README.md

```diff
@@ -4,6 +4,7 @@ SublimeSwitchFileDeluxe
 Switch header/source based on generic suffixes (not just the extension).
 
 If a file can't be found in the same directory, the command also searches for it in the directory of other already opened files.
+It can also search in relative paths specified with `"paths"` setting. For example if `"paths"` is `["..", "../impl", "internal"]`, then header/source will be searched in parent folder, in 'impl' sibling folder and in 'internal' subfolder.
```

Comment:
Since using this plugin usually implies copy-pasting-and-adjusting the doc example into a key mapping, could you add `"paths": ["../include"]` or something like that in the example below so that your feature can be easily discovered?